File: Fundamentals/Data Types and Variables - Exercise/gladiator_expenses.py
lost_fights_count = int(input())
helmet_price = float(input())
sword_price = float(input())
shield_price = float(input())
armor_price = float(input())
time_broken = {
    'helmet': 0,
    'sword': 0,
    'shield': 0,
    'armor': 0
}
for fight in range(1, lost_fights_count+1):
    if fight % 2 == 0:
        time_broken['helmet'] += 1
    if fight % 3 == 0:
        time_broken['sword'] += 1
    if fight % 2 == 0 and fight % 3 == 0:
        time_broken['shield'] += 1
        if time_broken['shield'] % 2 == 0:
            time_broken['armor'] += 1
total_amount = time_broken['helmet'] * helmet_price + time_broken['sword'] * sword_price + time_broken['shield'] * shield_price + time_broken['armor'] * armor_price
print(f'Gladiator expenses: {total_amount:.2f} aureus')

File: JS Front End/JS Syntax Fundamentals - More Exercises/gladiatorExpenses.js
function gladiatorExpenses(lostFightCount, helmetPrice, swordPrice, shieldPrice, armorPrice) {
    const items = {
        helmet: {times: 0},
        sword: {times: 0},
        shield: {times: 0},
        armor: {times: 0},
    }

    for (let fight = 1; fight <= lostFightCount; fight++) {
        if (fight % 2 === 0) {
            items.helmet.times += 1;
        }

        if (fight % 3 === 0) {
            items.sword.times += 1;
        }

        if (fight % 2 === 0 && fight % 3 === 0) {
            items.shield.times += 1;

            if (items.shield.times % 2 === 0) {
                items.armor.times += 1;
            }
        }


    }

    const result = items.helmet.times * helmetPrice + items.sword.times * swordPrice +
        items.shield.times * shieldPrice + items.armor.times * armorPrice;

    console.log(`Gladiator expenses: ${result.toFixed(2)} aureus`);
}

gladiatorExpenses(7, 2, 3, 4, 5);
gladiatorExpenses(23,
    12.50,
    21.50,
    40,
    200);